ScanDemoExample/scanlibrary/src/main/java/com/scanlibrary/PickImageFragment.java

```diff
@@ -10,7 +10,7 @@
 import android.os.Build;
 import android.os.Bundle;
 import android.provider.MediaStore;
-import android.support.v4.content.FileProvider;
+import androidx.core.content.FileProvider;
```

Comment:
Actually I tried to do this but while saving the image its crashing

Reply:
for me after clicking pic it shows black screen with folder and camera option it's not directing to crop and filters